src/routes/Events/components/EventsPage/EventsPage.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react'
 import { translate } from 'react-i18next'
 import PropTypes from 'prop-types'
-import DocumentTitle from 'react-document-title'
+import { Helmet } from 'react-helmet'
 
 import Event from '../Event'
 
 import styles from './EventsPage.scss'
 
 const EventsPage = ({ pastEvents, t }) => (
-  <DocumentTitle title={t('EventsPage.documentTitle')}>
+  <div>
```

Comment:
Two solution:

1. Remove unnecessary `div` and put `Helmet` inside
2.
```
<Helmet>
    {children}
</Helmet>
```

what do you prefer?